src/explore-education-statistics-frontend/server.js

```diff
@@ -84,32 +84,62 @@ async function startServer() {
     return `${input.slice(0, -pattern.length)}${replacement}`;
   }
 
+  function isRedirectionRequired(request) {
+    const hostnameIncludesWWW = request.hostname.startsWith('www');
+    let redirectionPath = request.path;
+
+    // Redirect URLs with trailing slash to equivalent without slash with 301
+    redirectionPath = replaceLastOccurrence(redirectionPath, '/', '');
+
+    // Bots are adding /1000 and Google isn't yet omitting them from its index automatically,
+    // so we should redirect away from them
+    redirectionPath = replaceLastOccurrence(redirectionPath, '/1000', '');
+
+    // Search wrongly indexed pages from Google Search Console for matches
+    // Redirect away if found to (eventually) clear routes from index
+    const urlMatch = seoRedirects.find(source => source.from === request.url);
```

Comment:
Not currently, no. The redirects in `redirects.js` are more or less as they come from Michael, so haven't had any previous rules applied and might still have trailing slashes or `/1000` on the end. 

I've been constantly going back and forth on how much sanitisation to do and where to do it. Atm I'm URL-encoding them and removing `https://explore-education-statistics` from the front, so we can match the paths, but as you point out I could apply the trailing slash and /1000 rules to them first as well. Some of the redirects from Michael have url params on them too, so need to think about that. 

I was originally trying to keep them as close to the source file as possible to avoid excessive manual steps, but think I'm leaning the other way now. Will re-assess in my next PR where I'm adding more redirects to the pile anyway. 

(Happy to rename the lambda to `seoRedirectPath` though :P)